scripts/apps/desks/controllers/DeskConfigController.ts

```diff
@@ -27,6 +27,7 @@ export function DeskConfigController($scope, $controller, notify, desks, modal)
     const initializeDesk = function(desk, step) {
         $scope.modalActive = true;
         $scope.step.current = step;
+        desk.desk_metadata = desk.desk_metadata ?? {};
```

Comment:
Why not initialize it in desk service instead?

Reply:
yep that works too, makes more sense, will update